src/main/webapp/index.html

```diff
@@ -39,23 +39,32 @@
     <div id="right-tab">
       <div id="pac-card">
         <h1>EatGoWhere</h1>
+        <a id="resize-nav-button">Collapse</a>
         <input id="pac-input" type="text" placeholder="Enter your location" />
+        <button id="submit-button">Submit</button>
         <div id="pac-container">
             <form id="restaurant-filter">
               <fieldset>
                 <legend>Dietery Restrictions</legend>
-                <input type="radio" name="restaurant-filter" id="halal" value="halal">
-                <label for="halal">Halal</label>
-                <input type="radio" name="restaurant-filter" id="vegetarian" value="vegetarian">
-                <label for="vegetarian">Vegetarian</label>
-                <input type="radio" name="restaurant-filter" id="vegan" value="vegan">
-                <label for="vegan">Vegan</label>
-                <input type="radio" name="restaurant-filter" id="gluten-free" value="gluten-free">
-                <label for="gluten-free">Gluten Free</label>
+                <div>
```

Comment:
Why did you make this change of adding div ?

Reply:
To prevent cases whereby the button and the legend are on different lines.

eg: 
![image](https://user-images.githubusercontent.com/30533810/88077353-c306b980-cbad-11ea-9226-ab1f080ea629.png)

Reply:
oh! so now it's line by line? if you want to have the div to still be side by side, while keeping the button and legend together, you can use `display: inline-block`

Reply:
What do you think works better? I was thinking side by side would be good for desktop since we have lots of horizontal screen space, but then in mobile it just looks messy.

Reply:
Yeah I agree that side by side looks good for desktop. Neutral for mobile!